src/components/Card/Card.tsx

```diff
@@ -1,58 +1,215 @@
+  /** Optional className that would be applied to the `card-actions` element */
   className?: string;
+  /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+  modifiers?: string[];
+  /** Optional boolean value to control visibility of border on the top edge of the card actions element */
+  topBorder?: boolean;
+}
+
+export interface CardProps {
+  /** Optional hex color value used to set the card background color */
+  backgroundColor?: string;
   /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
   blockName?: string;
-  /** Calls to Action for the card, such as "download" on a card for a book */
-  ctas?: React.ReactNode;
-  /** Footer that appears within the card but below the three columns within it */
-  footer?: React.ReactNode;
-  /** Heading that appears within the card but above the three columns within it */
-  heading?: React.ReactNode;
+  /** Optional boolean value to control the visibility of a border around the card */
+  border?: boolean;
+  /** Optional boolean value to control the alignment of the text and elements within the card */
+  center?: boolean;
+  /** ClassName that appears in addition to "card" */
+  className?: string;
+  /** Optional hex color value used to override the default text color */
+  foregroundColor?: string;
   /** ID that other components can cross reference for accessibility purposes */
   id?: string;
-  /** First column within the card */
-  image?: React.ReactNode;
+  /** Optional value to control the aspect ratio of the cartd image; default value is `square` */
+  // imageAspectRatio?: CardImageRatios;
+  /** Optional value to control how the image is sized image */
+  // imageSize?: CardImageSizes;
+  /** Optional boolean value to control the position of the image within the card */
+  imageFlip?: boolean;
+  /** Optional value to control the position of the image placeholder; default value is `vertical` */
+  layout?: CardLayouts;
   /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
   modifiers?: string[];
+  /** Optional padding value.  This value should be entered with the same formatting as a CSS padding attribute (ex. `5%`, `20px`, `2rem`).  If omitted, the Card will use the default padding. */
+  padding?: string;
+}
+
+// CardImage child-component
+export function CardImage(props: React.PropsWithChildren<CardImageProps>) {
+  const { src, alt, className, imageAspectRatio, imageSize, component } = props;
+  const classNames = ["image-wrap"];
+  imageAspectRatio && classNames.push(`image-wrap--` + imageAspectRatio);
+  // imageSize && classNames.push(`image-wrap--$(imageAspectRatio)`);
+  // <div className={`image-wrap image-wrap--${imageAspectRatio}`}></div>
+  return (
+    (src || component) && (
+      <>
+        <div className={bem("image", [imageSize], "card", [])}>
```

Comment:
This needs to be updated because right now `imageSize` is undefined but the array is an empty array. So this spits out `card__image--undefined`.
<img width="325" alt="Screen Shot 2021-08-18 at 12 18 34 PM" src="https://user-images.githubusercontent.com/1280564/129934665-739b2de2-1e25-40a1-9348-416cb9c7026a.png">

Reply:
Done.